### ISSUE TITLE
Link JAXB documentation instead of inlining it

### ISSUE THREAD

**Issue by zdila** [FroMage/jax-doclets#29]
Please add support for -link also for multiple JAXB projects - when one JAXB annotated class refers other from different project. Currently the referenced classes are in-lined.

**Comment by FroMage** [comment]
That's a good idea. Would you like to contribute to this?

**Comment by zdila** [comment]
Yes, if I had time. I'll try to check the source maybe next week or so.

**Comment by FroMage** [comment]
That's be great! Let me know if you need any help :)
